app/public/cantusdata/settings.py

```diff
@@ -132,7 +132,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/dev/howto/static-files/
 
-STATIC_URL = "/static/"
+STATIC_URL = "/static/django/"
```

Comment:
is this necessary? Does this mean that the paths for all static files will change from `cantus.simssa.ca/static/filename.jpg` -> `cantus.simssa.ca/static/django/filename.jpg`?

Reply:
> is this necessary?

It's possible I could keep it the same and modify the nginx configuration.

Reply:
I guess it doesn't matter too much, but if we're serving a file called `logo.png` or something, I think the path `/static/logo.png` makes more sense - It's a `static/` file called `logo.png`. Otherwise, `/static/django/logo.png` implies that it's a `static/` file (good); not only is it a static file but within that it's a `django/` (!?), which is called `logo.png`.

Thinking of writing up clear URIs probably matters less when we're serving images and bits of JS than when we're serving pages that represent things like manuscripts or chants, so if this would lead to significantly more complexity for developers, I think this is good as it is.

Reply:
Having thought about this more, I agree. I don't like this approach -- better, I think, to have nginx handle the figuring out of where to look for things than exposing this (really rather arbitrary) division between "static files collected by django" and "static files we've decided to make part of the nginx container".

Reply:
which, just to loop back to Andrew's point, is a fiction that is entirely Docker's fault